### ISSUE TITLE
0 Boxes detected always

### ISSUE THREAD

**Issue by kubasienki** [experiencor/keras-yolo2#308]
Hi,
I'have a problem with training network with my own dataset. I have even tried with white blobs on black background and still netowk is unable to detect anything.

`{
    "model" : {
        "backend":              "Full Yolo",
        "input_size":           416,
        "anchors":              [0.29,0.11, 1.83,0.57, 3.79,1.31, 7.82,2.63, 13.70,5.09],
        "max_box_per_image":    10,
        "labels":               ["Rock"]
    },

    "train": {
        "train_image_folder":   "----",
        "train_annot_folder":   "----",

        "train_times":          3,
        "pretrained_weights":   "full_yolo_racoon.h5",
        "batch_size":           16,
        "learning_rate":        1e-4,
        "nb_epochs":            50,
        "warmup_epochs":        3,

        "object_scale":         1.0 , (or 5.0 doesn't matter)
        "no_object_scale":      1.0,
        "coord_scale":          1.0,
        "class_scale":          1.0,

        "saved_weights_name":   "full_yolo_rock_2.h5",
        "debug":                true
    },

    "valid": {
        "valid_image_folder":   "",
        "valid_annot_folder":   "",

        "valid_times":          1
    }
}`

**Comment by kubasienki** [comment]
Just found out that if I stop training when loss in relatively high it detects objects but a lot of boxes at one object

**Comment by getsanjeev** [comment]
@kubasienki Have you found any solution?

**Comment by kubasienki** [comment]
Probably. My guess is that yolo architecture works on grid for which my objects are too small. But that's just my guess.
